autogpts/autogpt/pyproject.toml

```diff
@@ -23,8 +23,8 @@ serve = "autogpt.app.cli:serve"
 [tool.poetry.dependencies]
 python = "^3.10"
 anthropic = "^0.25.1"
-# autogpt-forge = { path = "../forge" }
-autogpt-forge = {git = "https://github.com/Significant-Gravitas/AutoGPT.git", subdirectory = "autogpts/forge"}
+autogpt-forge = { path = "../forge", develop = true }
+# autogpt-forge = {git = "https://github.com/Significant-Gravitas/AutoGPT.git", subdirectory = "autogpts/forge"}
```

Comment:
I'm aware this needs to be reverted before merging

Reply:
is this something we are ready to do?

Reply:
If we want to change it back to the git reference, we need to merge first (creating commit A), and then in a follow-up commit we can update the `autogpt-forge` dependency back to the git reference and lock it at commit A.

This is necessary because the git reference is locked to the *last* commit, and it's impossible to reference a commit from within the commit itself.